controllers/etcd_custodian_controller.go

```diff
@@ -168,6 +176,92 @@ func (ec *EtcdCustodian) updateEtcdStatus(ctx context.Context, logger logr.Logge
 	})
 }
 
+func calculatePDBminAvailable(etcd *druidv1alpha1.Etcd) int {
+	// do not enable in single mode
+	if etcd.Spec.Replicas < 2 {
+		return 0
+	}
+
+	// cluster is in bootstrap
+	if etcd.Status.ClusterSize == nil {
+		return 0
+	}
+
+	allMembersReady := false
+	backupReady := false
+	for _, condition := range etcd.Status.Conditions {
+		if condition.Type == druidv1alpha1.ConditionTypeAllMembersReady &&
+			condition.Status == druidv1alpha1.ConditionStatus(druidv1alpha1.EtcdMemberStatusReady) {
+			allMembersReady = true
+			continue
+		}
+
+		if condition.Type == druidv1alpha1.ConditionTypeBackupReady &&
+			condition.Status == druidv1alpha1.ConditionTrue {
+			backupReady = true
+			continue
+		}
+	}
+
+	clusterSize := int(*etcd.Status.ClusterSize)
+	values := make([]int, int(math.Floor(float64(clusterSize)/float64(2))+1))
```

Comment:
Hm, you might want to check this line 👀 I rather expected `int(math.Floor(float64(clusterSize)/float64(2))+1)` to be added to the values slice but instead it's used as `len`?